pkg/cache/queue/cluster_queue.go

```diff
@@ -63,6 +63,14 @@ type ClusterQueue struct {
 	namespaceSelector labels.Selector
 	active            bool
 
+	// pendingWorkloadCount is the number of workloads in the ClusterQueue that
+	// are in a pending state.
+	pendingWorkloadCount int
+
+	// runningWorkloadCount is the number of workloads in the ClusterQueue that
+	// are in a running state (i.e. have the WorkloadPodsReady condition set to True).
+	runningWorkloadCount int
```

Comment:
Do we need to record them in state? Maybe CountWorkloads() (int, int) could just return them instead?